step.sh

```diff
@@ -3,10 +3,10 @@ set -ex
 
 if [[ $OSTYPE == 'darwin'* ]]; then
   echo 'macOS agents preinstall marathon-cloud cli via brew'
-  marathon-cloud -api-key "$api_key" -apk "$BITRISE_APK_PATH" -testapk "$BITRISE_TEST_APK_PATH"
```

Comment:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#never-depend-on-environment-variables-in-your-step
Here, there is a recommendation to avoid the usage of envs in your scripts.